src/tests/ftest/aggregation/dfuse_space_check.py

```diff
@@ -106,13 +110,14 @@ def test_dfusespacecheck(self):
         # Create a pool, container, and start dfuse
         self.create_pool()
         self.create_cont()
-        self.start_dfuse(self.hostlist_clients, self.pool, self.container)
+        dfuse = get_dfuse(self, self.hostlist_clients)
+        start_dfuse(self, dfuse, self.pool, self.container)
```

Comment:
Sorry to ask such a high level question when you're so close to getting this working but why is `start_dfuse` a function, not a method on the DFuse class?
```suggestion
        dfuse = get_dfuse(self, self.hostlist_clients)
       dfuse.start(self.pool, self.container)
```

Reply:
The start_dfuse method also registers the method to be called during tearDown that will stop dfuse.

Reply:
It could, but if we're making a interface change that affects the entire suite isn't it better to make that change once rather than planning a second wholesale change as well?

Reply:
I see what you mean. It depends on how complicated it is now vs later. I'm talking with @phender about this now.

Reply:
I obviously don't know the details but overall one change is better than two.  I am really glad to see this kind of work progressing though.